x-pack/plugins/apm/server/routes/services/get_services/get_service_aggregated_transaction_stats.ts

```diff
@@ -80,6 +80,11 @@ export async function getServiceAggregatedTransactionStats({
                   size: maxNumServices,
                 },
                 aggs: {
+                  overflowCount: {
+                    max: {
+                      field: 'service.aggregation.overflow_count',
```

Comment:
Can you make this a constant and add it to https://github.com/elastic/kibana/blob/main/x-pack/plugins/apm/common/es_fields/apm.ts ?